### PR TITLE
Added basic 3d extrusion example

### PR DESCRIPTION
**Pull request by langsmith**
@cammace , the commit history for the previous pr related to this basic extrusion example just got too messed up. I decided to re-do things with a fresh branch/pr

Addresses https://github.com/mapbox/mapbox-android-demo/issues/315
<br>

ezgif com-resize 2

### CODE REVIEW

File: MapboxAndroidDemo/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -387,14 +386,18 @@
...
         </activity>
-
         <activity
             android:name=".examples.camera.RestrictCameraActivity"
             android:label="@string/activity_camera_restrict_title">
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
         </activity>
+        <activity android:name=".examples.styles.BasicExtrusionActivity">
```
Comment (cammace) [inline MapboxAndroidDemo/src/main/AndroidManifest.xml]: Missing a `android:label="">` attribute

File: MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/BasicExtrusionActivity.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import com.mapbox.mapboxsdk.style.functions.stops.IdentityStops;
+import com.mapbox.mapboxsdk.style.layers.FillExtrusionLayer;
+
+import static com.mapbox.mapboxsdk.style.layers.Filter.eq;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillExtrusionBase;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillExtrusionColor;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillExtrusionHeight;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillExtrusionOpacity;
+
+
```
Comment (cammace) [inline MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/BasicExtrusionActivity.java]: nit

File: MapboxAndroidDemo/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -96,6 +97,7 @@
     <string name="activity_lab_las_angeles_tourism_title">Los Angeles for tourists</string>
     <string name="activity_lab_indoor_map_title">Indoor Map</string>
 
+
```
Comment (cammace) [inline MapboxAndroidDemo/src/main/res/values/strings.xml]: nit

**Comment by langsmith** [comment]
Cleaned up @cammace

**Pull request merged**